src/plugins/inputs/otr.js

```diff
@@ -0,0 +1,17 @@
+module.exports = function(network, chan, cmd, args) {
+	if (cmd !== "otr") {
+		return;
+	}
+	if (args.length === 0 || args[0] === "") {
+		return;
```

Comment:
Objectively, I would say yes, but looking around other `inputs` plugins, I see that badly-formed commands are just ignored :-( : [`/me`](https://github.com/astorije/shout/blob/02b28b4b497637331a452cee97a40c7a5483f567/src/plugins/inputs/action.js#L17), [`/connect`](https://github.com/astorije/shout/blob/02b28b4b497637331a452cee97a40c7a5483f567/src/plugins/inputs/connect.js#L5), [`/invite`](https://github.com/astorije/shout/blob/02b28b4b497637331a452cee97a40c7a5483f567/src/plugins/inputs/invite.js#L6), ... and literally all others!

I suggest to leave this PR as is, and consider https://github.com/astorije/shout/issues/11 when the overall PR is in good shape.